ReactiveUI/AddTo.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace System.Reactive.Disposables
+{
+    public static class CompositeDisposableExtensions
+    {
+        /// <summary>
+        /// Adds a disposable to the specified <see cref="CompositeDisposable"/>.
+        /// </summary>
+        /// <typeparam name="T">
+        /// The type of the disposable.
+        /// </typeparam>
+        /// <param name="this">
+        /// The disposable.
+        /// </param>
+        /// <param name="compositeDisposable">
+        /// The <see cref="CompositeDisposable"/> to which <paramref name="this"/> will be added.
+        /// </param>
+        /// <returns>
+        /// The disposable.
+        /// </returns>
+        public static T AddTo<T>(this T @this, CompositeDisposable compositeDisposable)
```

Comment:
what do you need to return the disposable for ? I'd except that once you put the disposable instance into the composite, re-using it for something else is probably not what you'd want ?